Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
       git-core \
       wget \
       curl && \
+    # ckan harvest plugin dependencies \
+    apt-get install --yes --no-install-recommends \
+      redis-server && \
```

Comment:
We don't want to install redis server inside the docker image.

Containers usually only run a single service. And we are already running our own CKAN instance. We already have a redis server running in the `redis` service of the docker-compose file. Lets use that.